Formula/h/hatch.rb

```diff
@@ -20,6 +20,7 @@ class Hatch < Formula
   depends_on "certifi"
   depends_on "cryptography"
   depends_on "python@3.12"
+  depends_on "uv"
```

Comment:
Not sure this is sufficient, because the `uv` PyPI package has `uv` Python bindings, but the formula does not.

Reply:
This should be fine, it checks the PATH: https://github.com/pypa/hatch/blob/hatch-v1.10.0/src/hatch/env/virtual.py#L104-L128

Reply:
I can confirm, as the maintainer. This should be good to merge!

Reply:
Thanks both!